Dockerfile-tests

```diff
@@ -0,0 +1,12 @@
+FROM hseeberger/scala-sbt
```

Comment:
You might consider something like https://hub.docker.com/r/gafiatulin/alpine-sbt/, or you could roll your own. Since the Dockerfiles are public, there's no need for an "official" one as long as you're ok with how it's built.